src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml

```diff
@@ -263,8 +265,8 @@ else
                 <div class="row">
                     <div class="col-md-12">
                         <div class="area_title">
-                            <h3>Zapisz się na newsletter</h3>
-                            <p>I nie przegap najważniejszych wydarzeń!</p>
+                            <h3>@SubscribeNewsletter</h3>
```

Comment:
please rename it to `SubscribeToNewsletter`

Reply:
done